test/onnx/test_fx_op_consistency.py

```diff
@@ -451,8 +467,8 @@
         "nn.functional.dropout",
         reason=onnx_test_common.reason_dynamo_does_not_support("Dropout"),
     ),
-    xfail(
-        "nn.functional.elu",
+    skip(
+        "nn.functional.elu",  # SegFault when ORT < 1.15
         dtypes=(torch.float64,),
         reason=onnx_test_common.reason_onnx_runtime_does_not_support("Elu", "float64"),
```

Comment:
We can use enable_if and check ort's version?

Reply:
The thing is that would be having this skip saying enabled if ORT < 1.15, and another xfail saying enabled_if ORT is 1.15

Reply:
I see so ort 1.15 still doesn't support the case, it just doesn't segfault?

Reply:
Yes